preferences/fedora/kustomization.yaml

```
@@ -3,26 +3,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - ../base
-
-components:
-  - ./metadata
-  - ./requirements
-  - ../components/diskbus-virtio-blk
-  - ../components/interfacemodel-virtio-net
-  - ../components/rng
-  - ../components/secureboot
-
-patches:
-  - target:
-      kind: VirtualMachinePreference
-    patch: |-
-      - op: replace
-        path: /metadata/name
-        value: fedora
-  - target:
-      kind: VirtualMachineClusterPreference
-    patch: |-
-      - op: replace
-        path: /metadata/name
-        value: fedora
+  - ./arm64
```

```suggestion
  - ./aarch64
```

Use the equivalent to `amd64`?